srcpkgs/nautilus-python/template

```diff
@@ -3,16 +3,17 @@ pkgname=nautilus-python
 version=1.2.3
 revision=3
 build_style=gnu-configure
-configure_args="--enable-gtk-doc PYTHON=python3"
-hostmakedepends="automake libtool pkg-config which"
-makedepends="python3-devel python3-gobject-devel gtk-doc nautilus-devel"
+configure_args="--disable-gtk-doc PYTHON=python3
```

Comment:
When the configure_args changed, please bump revision.
@mnabid Do you still want to have gtk-doc? Should we add a build_options instead?